pkg/server/server.go

```diff
@@ -6,7 +6,9 @@ package server
 import (
 	"crypto/tls"
 	"crypto/x509"
+	"encoding/pem"
 	"fmt"
+	uuid2 "github.com/lf-edge/eve/api/go/uuid"
```

Comment:
This looks like it is in the wrong place. It should go around L16, with the other `api/go/` entries.

Reply:
I know, it is just stylistic, but it is good to be consistent.

Reply:
Fixed